translations/messages.en.yml

```diff
@@ -1284,6 +1285,7 @@ adminDashboard.nonprofits.confirm_delete: Are you sure you want to delete this n
 order.cart.donation.disclaimer: We offer you to donate %percentage% of the revenue from your order to one of our partner associations. You will not pay more.
 form.checkout_address.nonprofit.placeholder: Choose your favorite nonprofit
 form.delivery.arbitrary_price.label: Define the price manually
+form.delivery.bookmark.label: Bookmark
```

Comment:
Don't forget to add french, spanish, italian, german translations at least! - we use chatgpt to do that with Antoine